test/house_test.rb

```diff
@@ -0,0 +1,135 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+  def test_a_house_exists
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+  end
+
+  def test_house_has_a_price
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal 400000, house.price
+  end
+
+  def test_house_has_an_address
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal "123 sugar lane", house.address
+  end
+
+  def test_house_starts_with_no_rooms
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal [], house.rooms
+  end
+
+  def test_that_you_can_add_rooms
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+    house.add_room(room_1)
+    house.add_room(room_2)
+    assert_equal 2, house.rooms.count
+  end
+
+  def test_house_starts_below_market_average
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal false, house.above_market_average?
+  end
```

Comment:
Any time a method could return true or false, its best to confirm both outcomes.